frontend/package.json

```diff
@@ -12,38 +12,38 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/common": "^4.0.0",
-    "@angular/compiler": "^4.0.0",
-    "@angular/core": "^4.0.0",
-    "@angular/forms": "^4.0.0",
-    "@angular/http": "^4.0.0",
-    "@angular/platform-browser": "^4.0.0",
-    "@angular/platform-browser-dynamic": "^4.0.0",
-    "@angular/router": "^4.0.0",
-    "@ng-bootstrap/ng-bootstrap": "^1.0.0-beta.1",
-    "bootstrap": "^4.0.0-beta",
+    "@angular/common": "^5.0.1",
+    "@angular/compiler": "^5.0.1",
+    "@angular/core": "^5.0.1",
+    "@angular/forms": "^5.0.1",
+    "@angular/http": "^5.0.1",
+    "@angular/platform-browser": "^5.0.1",
+    "@angular/platform-browser-dynamic": "^5.0.1",
+    "@angular/router": "^5.0.1",
+    "@ng-bootstrap/ng-bootstrap": "^1.0.0-beta.5",
+    "bootstrap": "^4.0.0-beta.2",
```

Comment:
Please double check if we need `bootstrap` at all since we use `@ng-bootstrap` (I'm not sure, but I kind of doubt).